app/views/draft_mailer/draft_published_notification.text.erb

```diff
@@ -1,5 +1,8 @@
 <%= @user[:name] %>,
-
+<% env = Rails.configuration.cmr_env.upcase %>
```

Comment:
I think you can use the change I suggested above and not need the if/else for the main content. But if it is necessary, I think it would be better to indent what is after the if and else.